### ISSUE TITLE
having trouble finding what i need for F-16C (ANT ELEV, MAN RNG, etc.) throttle stuff.

### ISSUE THREAD

**Issue by Leavvv** [dcs-bios/dcs-bios#269]
So, i am making a DIY throttle. Issue is, i cannot find all of the switches i need to make it. Is using DCS-Bios the way to go with this? this is my first DIYHotas project but I thought I would ask here first.

thank you and much love

**Comment by WarLord211** [comment]
Hotas Buttons usualy noch clickable with the mouse. So the are "not" in the code of the module/in the part BIOS works.

so i cant program them because i dint find there valves

**Comment by jgwheeler** [comment]
DCS-BIOS is not really meant for what you are doing, which is just building a custom game/HID controller. There are lots of hardware solutions out there that provide a ready-to-go USB HID controller where you just wire up your buttons, knobs, sensors, etc. and then you assign functions to the buttons/axes in game just like you would with any other joystick/throttle.

Example: https://www.leobodnar.com/shop/index.php?main_page=product_info&cPath=94&products_id=204&zenid=1ff71c972d84da8e17cd45b5bb98c4be

**Comment by Leavvv** [comment]
yeah, i figured a while ago but thank you! i actually have already
completed the throttle and use it on DCS regularly now :)

**Comment by skyriderfpv** [comment]
HI, how did you figure it out?, i´m interested in the solution for the same purposes but with an arduino UNO, since i hgave a couople lying around...

**Comment by Leavvv** [comment]
wow this was like a couple years ago almost. So yeah, turns out DCS-bios
isnt what you need to create a throttle. As long as you have a USB
compatible arduino (ATmega32U4), then you can just use a rotary encoder or
a dual potentiometer. I failed with this project when I tried creating a
common ground for everything, but it did work! Hope this helps.

**Comment by skyriderfpv** [comment]
Thank you. I got a pro micro with the ATMega32U4 and it's working very nice. The prototype it's working. Thank you for the advise.
Right now I'm improving details for precision and better feel with the Throttle and, but everything working. I'll be posting a repository with the project.

**Comment by Leavvv** [comment]
Glad to hear it!
